sites/svelte.dev/src/routes/docs/+server.js

```diff
@@ -0,0 +1,13 @@
+// @ts-expect-error custom suffix doesn't have types
```

Comment:
this shouldn't be `@ts-expect-error` because that means the problem is unsolvable. if we have to use an annotation here it should be `@ts-ignore`, which means that we're just too lazy to fix it at the moment :laughing: 

Vite does this by providing query strings like `?raw` and then defining types for them. I think we could do the same with a query string like `?minified`